### PR TITLE
Fixes the side navigation bar

### PR DESCRIPTION
**Pull request by harsh253**
**Problem**
To fix the position of navigation bar so that it does not scroll as the user scrolls down the main content

**Issue this PR referred to**
Fixes issue number 328. Refer to https://github.com/codeuino/social-platform-donut-frontend/issues/328 for more details.

**Solution of problem**
I have fixed the height of the body element to 100% and allowed scrolling of all content of the page except that of navigation bar. Also made some changes in the display property of main content.

**Before changes were made**
Hnet com-image

**After changes were made**
solution

### CODE REVIEW

File: src/user/dashboard/news-feed/news-feed.js [hunk 1]
```diff
@@ -1,11 +1,239 @@
...
+        <ul className="list-unstyled">
+          {children}
+        </ul>
+      </div>
+    );
+  },
+);
+
+const feed = [
+  {
```
Comment (vaibhavdaren) [inline src/user/dashboard/news-feed/news-feed.js]: can you load feed array from a separate file  named `json_data`

Comment (harsh253) [reply src/user/dashboard/news-feed/news-feed.js]: Sure, I'll work on this. 
Thanks for the feedback.

Comment (harsh253) [reply src/user/dashboard/news-feed/news-feed.js]: Also , if you could let know any other changes that may be required.

Comment (elit-altum) [reply src/user/dashboard/news-feed/news-feed.js]: This is good. Squash your commits as well, when you are done

Comment (harsh253) [reply src/user/dashboard/news-feed/news-feed.js]: Thanks @elit-altum. How do I squash my commits ? Also if you could let me know why is it required ?

**Pull request closed without merging**